### PR TITLE
Fixed Pre-Release PR workflow version bumping

### PR DESCRIPTION
**Pull request by samiib**
- Fixed the version bumping in git-cliff
- Fixed typos from spell checking ACI and other acronyms

### CODE REVIEW

File: .github/workflows/release_pr.yml [hunk 1]
```diff
@@ -14,11 +14,48 @@ jobs:
...
+          esac
+
+      # Commit major or minor version change messages
+      # NOTE: Without these commit messages git-cliff will only increase patch version.
+      - name: Commit major version change message
+        if: steps.type.outputs.bump == 'MAJOR'
+        run: 'git commit -m "major: [ignore] Major version change" --allow-empty'
+      - name: Commit minor version change message
+        if: steps.type.outputs.bump == 'MINOR'
+        run: 'git commit -m "minor: [ignore] Minor version change" --allow-empty'
```
Comment (lhercot) [inline .github/workflows/release_pr.yml:56]: Is this the only way?

**Review by lhercot — CHANGES_REQUESTED** [review]
Seems you started this branch from the old one and has older commits.

**Comment by samiib** [comment]
> Seems you started this branch from the old one and has older commits.

This is because you broke the commit history by using the abomination of a feature called "squash merging". 😞 
I rebased my branch off master after you merged but now I will need to manually modify my own branch history to fix it. 
This is another reason why I don't think we should squash merge.

File: .github/workflows/release_pr.yml (hunk 1 above)
Comment (samiib) [reply .github/workflows/release_pr.yml]: I found a way to dynamically insert a commit message into git-cliff without it being in the commit history using the --with-commit argument.
Looks like the most suitable way for the tool to register a major/minor version bump without an empty commit.

**Comment by lhercot** [comment]
I have never had an issue rebasing when squash merging and I had to do it to fix the commit message issues...

**Comment by samiib** [comment]
> I have never had an issue rebasing when squash merging and I had to do it to fix the commit message issues...

That's, okay the commit messages have been changed. 👍 https://github.com/CiscoDevNet/terraform-provider-aci/pull/1256/commits
@lhercot Let me know if you have any other concerns?

**Review by anvitha-jain — COMMENTED** [review]
LGTM

**Review by anvitha-jain — APPROVED** [review]

**Review by gmicol — APPROVED** [review]
LGTM

**Review by lhercot — APPROVED** [review]
LGTM

**Pull request merged**